pkgs/development/php-packages/couchbase/default.nix

```diff
@@ -1,34 +1,37 @@
-{ lib, buildPecl, fetchFromGitHub, writeText, libcouchbase, zlib, php, substituteAll }:
+{ lib, buildPecl, fetchFromGitHub, zlib, cmake, pkg-config, openssl_1_1, php }:
 let
+  version = "4.1.4";
   pname = "couchbase";
-  version = "3.2.2";
 in
 buildPecl {
   inherit pname version;
 
   src = fetchFromGitHub {
     owner = "couchbase";
-    repo = "php-couchbase";
-    rev = "v${version}";
-    sha256 = "sha256-JpzLR4NcyShl2VTivj+15iAsTTsZmdMIdZYc3dLCbIA=";
+    repo = "couchbase-php-client";
+    rev = version;
+    fetchSubmodules = true;
+    hash = "sha256-twu3zYhSCAvjZJl/fp4vzkE4PHw23QzCkfEt6ZOIRtw=";
   };
 
-  configureFlags = [ "--with-couchbase" ];
+  dontUseCmakeConfigure = true;
 
-  buildInputs = [ libcouchbase zlib ];
+  env.COUCHBASE_CMAKE_EXTRA = "-DCOUCHBASE_CXX_CLIENT_EMBED_MOZILLA_CA_BUNDLE=false -DCMAKE_SKIP_BUILD_RPATH=true -DCMAKE_SKIP_INSTALL_RPATH=true";
```

Comment:
We do not want `-DCMAKE_SKIP_BUILD_RPATH=true` but can be used as a work around.

We should not need `-DCMAKE_SKIP_INSTALL_RPATH=true`.